Suomalaiset käännökset/ui.json

```diff
@@ -250,16 +250,16 @@
   "CharacterOption": {
     "TEXT": [
       "Lukittu",
-      "Tämä hahmo avautuu suorittamalla yhden pelikerran.",
+      "Tämä hahmo avautuu läpäisemällä yhden pelikerran.",
```

Comment:
Pelin läpäisyä ei vaadita kolmen ensimmäisen hahmon kohdalla, riittää että pelaa yhden pelin.
Vertaa seuraavia:
"This character is unlocked by **completing** a single run."
"This character is unlocked by **winning** a Standard run and having the 3rd character unlocked."

Reply:
Oho, niinpä taitaa ollakin.